app/admin/assets/css/admin.css

```diff
@@ -16414,6 +16414,10 @@ a.text-white:hover, a.panel-white:hover, a.text-white:focus, a.panel-white:focus
 .text-muted {
   color: #9194A6 !important;
 }
+.text-draggable {
+  -webkit-user-select: all; /* for Safari */
+  user-select: all;
+}
```

Comment:
This should go in `app/system/assets/ui/scss/utilities/_utilities.scss` and add `cursor: move;`